lib/fog/openstack/models/identity_v3/tokens.rb

```diff
@@ -10,6 +10,7 @@ class Tokens < Fog::OpenStack::Collection
 
 
           def authenticate(auth)
+            @@cache ||= {}
```

Comment:
@dhague right, the question was about, that I don't see this cache used anywhere?